server/svix-server/src/openapi.rs

```diff
@@ -41,7 +43,7 @@ pub fn initialize_openapi() -> OpenApi {
     OpenApi {
         info: openapi::Info {
             title: "Svix API".to_owned(),
-            version: "1.4".to_owned(),
+            version: VERSION.to_owned(),
```

Comment:
Huh, interesting.

Reply:
And, by "interesting" I mean I wonder if the version bump script previously used to increment this, but no more, or what the breakage might have been.